test-utils/src/index.js

```diff
@@ -31,3 +31,35 @@ export function act(cb) {
 	options.debounceRendering = Component.__test__previousDebounce;
 	options.requestAnimationFrame = previousRequestAnimationFrame;
 }
+
+/**
+ * Setup the test environment
+ * @returns {HTMLDivElement}
+ */
+export function setupScratch() {
```

Comment:
Hmm I think I'd prefer we don't export a `setupScratch` helper in test-utils. I don't think it adds much value specifically for Preact testing, of which there are many approaches only some of which include adding a DOM. While it is very useful for how we do testing internally in the Preact repo, it isn't necessarily useful for users of Preact.

In other words, I don't think this function adds something unique about testing in Preact to justify its maintenance as a public api.

(Note: Personally, I like to keep exposed APIs to a minimum to reduce long term baggage, but perhaps I'm overreacting here 😅)

Reply:
Well in all honesty, I don't really mind it was more or less to give a clearcut way for people to test basic stuff.
That's also one thing we can leave out of teardown then